### PR TITLE
Update ht to 2.1.0

### PR DESCRIPTION
**Pull request by Midar**

### CODE REVIEW

File: Library/Formula/ht.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 class Ht < Formula
   homepage 'http://hte.sf.net/'
-  url 'https://downloads.sourceforge.net/project/hte/ht-source/ht-2.0.22.tar.bz2'
-  sha256 'c729d64bf7de440c7b1021d3d6657ccbdb103541b4082a58dca7c8402c773f58'
+  url 'https://sourceforge.net/projects/hte/files/ht-source/ht-2.1.0.tar.bz2'
```
Comment (jacknagel) [inline Library/Formula/ht.rb]: Please preserve the `downloads.sourceforge.net` URL

Comment (Midar) [inline Library/Formula/ht.rb]: Amended.

**Comment by mikemcquaid** [comment]
If there's any chance you could fix up the `brew audit --strict ht` that'd be amazing. If not, no worries and I'll merge this.

**Comment by Midar** [comment]
Woah, I just tried it and it would start downloading Ruby gems without asking me. Are these even checked for signature or this basically arbitrary remote code execution? There really should be a confirmation before installing things, especially as without -v it doesn't even tell you it's installing things in the background! I only noticed because it took a long while, I canceled and retried with -v.

**Comment by Midar** [comment]
Fixed the issue using the build bot output now.

**Comment by mikemcquaid** [comment]
I don't see a new commit here?

**Comment by Midar** [comment]
Oh, apparently forgot to push. Sorry, fixed.

File: Library/Formula/ht.rb [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
-  url 'https://downloads.sourceforge.net/project/hte/ht-source/ht-2.0.22.tar.bz2'
-  sha256 'c729d64bf7de440c7b1021d3d6657ccbdb103541b4082a58dca7c8402c773f58'
+  url 'https://downloads.sourceforge.net/project/hte/ht-source/ht-2.1.0.tar.bz2'
+  sha256 '31f5e8e2ca7f85d40bb18ef518bf1a105a6f602918a0755bc649f3f407b75d70'
 
   depends_on 'lzo'
 
   def install
-    system "chmod +x ./install-sh"
+    FileUtils.chmod "+x", "./install-sh"
```
Comment (mikemcquaid) [inline Library/Formula/ht.rb]: Sorry, this can have the `FileUtils.` omitted because it's included into the `Formula` class.

Comment (Midar) [inline Library/Formula/ht.rb]: Ah, thanks. Updated.

File: Library/Formula/ht.rb [hunk 3]
```diff
@@ -2,13 +2,13 @@
...
-  url 'https://downloads.sourceforge.net/project/hte/ht-source/ht-2.0.22.tar.bz2'
-  sha256 'c729d64bf7de440c7b1021d3d6657ccbdb103541b4082a58dca7c8402c773f58'
+  url 'https://downloads.sourceforge.net/project/hte/ht-source/ht-2.1.0.tar.bz2'
+  sha256 '31f5e8e2ca7f85d40bb18ef518bf1a105a6f602918a0755bc649f3f407b75d70'
 
   depends_on 'lzo'
 
   def install
-    system "chmod +x ./install-sh"
+    chmod "+x", "./install-sh"
```
Comment (mikemcquaid) [inline Library/Formula/ht.rb]: Rargh, annoying this needs to be `chmod 0755, "./install-sh"` because of Mountain Lion's old Ruby. Sorry!

Comment (Midar) [inline Library/Formula/ht.rb]: Only tested on Yosemite, sorry. Fixed.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew and patience! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**